docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -210,7 +233,75 @@ data:
 EOF
 ----
 +
-The resources in the `ConfigMap` contain cell-specific configurations.
+
+. Create dataplane services for Nova cells to enable pre-upgrade workarounds,
+and to configure the {compute_service} services for a picked storage back end:
+
++
+[source,yaml]
+----
+for CELL in $(echo $RENAMED_CELLS); do
+  oc apply -f - <<EOF
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-$CELL
+spec:
+  dataSources: <1>
+    - secretRef:
+        name: nova-$CELL-compute-config <2>
+    - secretRef:
+        name: nova-migration-ssh-key <3>
+    - configMapRef:
+        name: nova-cells-global-config
+        optional: true
+  playbook: osp.edpm.nova
+  caCerts: combined-ca-bundle
+  edpmServiceType: nova
+  containerImageFields:
+  - NovaComputeImage
+  - EdpmIscsidImage
+EOF
+  done
+----
++
+
+* If TLS Everywhere is enabled, append the following content to the `OpenStackDataPlaneService` CR:
++
+[source,yaml]
+----
+  tlsCerts:
+    contents:
+      - dnsnames
+      - ips
+    networks:
+      - ctlplane
+    issuer: osp-rootca-issuer-internal
+  caCerts: combined-ca-bundle
+  edpmServiceType: nova
+----
++
+<1> To enable a local metadata services for a cell<N>, append a `spec.dataSources.secretRef` to reference
+an additional auto-generated `nova-cell<N>-metadata-neutron-config` secret. You should have also set
+`spec.nova.template.cellTemplates.cell<N>.metadataServiceTemplate.enable` in the `OpenStackControlPlane/openstack` CR.
+<2> The secret `nova-cell<N>-compute-config` auto-generates for each `cell<N>`.
+<3> You must append the `nova-cell<N>-compute-config` and `nova-migration-ssh-key` references for each custom `OpenStackDataPlaneService` CR that is related to the {compute_service}.
+
+* For simple configuration overrides, we do not need a custom dataplane service. However, to reconfigure the cell `cell1` in general,
+the safest option would be always creating a custom service, and a dedicated configuration map for it.
+
+[NOTE]
+The cell `cell1` is already managed with the default `OpenStackDataPlaneService` called `nova`
+and its `nova-extra-config` configuration map. Do not change the default dataplane service 'nova' definition.
+The changes will be lost, when the {rhos_long} operator becomes updated with OLM.
+
+* When a cell spans multiple node sets, you might want to name the custom `OpenStackDataPlaneService` resources like
+`nova-cell1-nfv` and `nova-cell1-enterprise`. Then the auto-generated configmaps would be named
+`nova-cell1-nfv-extra-config` and `nova-cell1-enterprise-extra-config`.
+
+[NOTE]
+Different configurations for nodes in multiple node sets of the same cell are not covered in this guide.
```

Comment:
I would add:
Different configurations for nodes in multiple node sets of the same cell are supported but not covered...